src/modules/sub/generate-youtube-tokens.js

```diff
@@ -0,0 +1,34 @@
+import { Innertube } from 'youtubei.js';
+
+const bail = (...msg) => {
+    console.error(...msg);
+    throw new Error(msg);
+};
+
+const tube = await Innertube.create();
+
+tube.session.once(
+    'auth-pending',
+    ({ verification_url, user_code }) => console.log(
+        `Open ${verification_url} in a browser and enter ${user_code} when asked for the code.`
+    )
+);
+
+tube.session.once('auth-error', (err) => bail('An error occurred:', err));
+tube.session.once('auth', ({ status, credentials, ...rest }) => {
+    if (status !== 'SUCCESS') {
+        bail('something went wrong', rest);
+    }
+
+    console.log(credentials);
+
+    console.log(
```

Comment:
i'd add a message like "add this cookie to youtube_oauth array in your cookies file:"